bin/constraints.py

```diff
@@ -105,39 +106,41 @@ def f(cs):
     left_width = 8 + name_len + 2 * value_len
 
     print('{:^{width}}|{:^30}'.format('VALIDATORS', '      PROBLEM STATEMENT', width=left_width),
-          sep='')
+          sep='',
+          file=sys.stderr)
```

Comment:
I would say that the `constraints` command should show its output on the standard output stream, as it falls in the category of "actual 'program output' and not progress/diagnostics" (https://github.com/RagnarGrootKoerkamp/BAPCtools/issues/92#issuecomment-751534349).

Reply:
works for me; i was in doubt for this one